resources/ark/values.yaml

```diff
@@ -71,5 +71,5 @@ global:
   containerRegistry:
```

Comment:
here I think we should also bump eu.gcr.io/kyma-project/etcd-tls-setup

Reply:
done